packages/mobile/src/identity/utils.test.ts

```diff
@@ -0,0 +1,22 @@
+import { extractShortSecurityCodeMessage, hashAddressToSingleDigit } from 'src/identity/utils'
+
+describe(hashAddressToSingleDigit, () => {
+  it('should compute correct hash', () => {
+    expect(hashAddressToSingleDigit('0x000000000000000000000008')).toEqual(8)
+    // 0xf7f551752A78Ce650385B58364225e5ec18D96cB -> 1415591498931780605110544902041322891412830525131
+    expect(hashAddressToSingleDigit('0xf7f551752A78Ce650385B58364225e5ec18D96cB')).toEqual(1)
+  })
+})
+
+describe(extractShortSecurityCodeMessage, () => {
+  it('should extract with correct message', () => {
+    expect(extractShortSecurityCodeMessage('<#> Celo security code: 51365977 5yaJvJcZt2P')).toEqual(
+      '51365977'
+    )
+  })
+  it('should NOT extract with incorrect message', () => {
```

Comment:
Sorry, I don't follow. Why is this an incorrect message when the one above is correct?

Reply:
The security code consist of two parts: first digit is the prefix we use to identify the validator, when we receive the code and the rest 7 digits is the code itself. I will a comment there with the explanation as well 👍